acmwebsite/controllers/root.py

```diff
@@ -49,6 +50,7 @@ class RootController(BaseController):
     schedule = ScheduleController()
     error = ErrorController()
     contact = ContactController()
+    projects = ProjectsController()
```

Comment:
Individual projects should probably be located at `acm.mines.edu/p/###` (users are `/u/***`, surveys are `/s/###`, etc). Is there a way to do that while keeping the `/projects` list page? Adding an additional `ProjectController`?

Reply:
This PR doesn't deal with individual project pages, so I think this is a consideration for #40. Personally, I don't like the short URLs. I think they are ugly. I've added this to the considerations section on #40.

Reply:
Let's continue use of short URLs even if Sumner does not like it... better for Emails, and looks better too.

Sorry Sumner.

Reply:
@jackrosenthal, fine by me. Still doesn't matter here, this PR only deals with the project listing page. From what I can see, we agree on this URL scheme:

- `/projects` - shows a list of all the projects with links to the individual projects
- `/p/<project_id>` - shows the project with id `project_id`

I think the real question is how to implement this. I think worst case scenario we have to add a `ProjectLookupController` and `ProjectController` in addition to the `ProjectsController`. Alternatively, we could expose the `/projects` endpoint on the root controller which would avoid that mess. My only concern with that is that it clutters up the root controller.

CC: @samsartor

Reply:
I was mainly thinking about future work on #40. Maybe we should eventually rename `ProjectsController` to `ProjectListController` so that it does not get confused with the incoming `ProjectController` and `Project[s|Lookup]Controller`. I'm feeling confused just listing them.